x-pack/plugins/observability_solution/apm/server/routes/dependencies/get_top_dependency_spans.ts

```diff
@@ -162,7 +161,7 @@ export async function getTopDependencySpans({
       duration: span.span.duration.us,
       traceId: span.trace.id,
       outcome: (span.event?.outcome || EventOutcome.unknown) as EventOutcome,
-      transactionId: span.transaction!.id,
+      transactionId: transaction?.transaction.id,
```

Comment:
Q: what's the difference between having `transaction.id` in `transaction` vs `span`?

Reply:
span documents for otel don't have `transaction.id`